gradoop-examples/gradoop-examples-operators/src/main/java/org/gradoop/examples/quickstart/data/QuickstartData.java

```diff
@@ -0,0 +1,40 @@
+package org.gradoop.examples.quickstart.data;
+
+/**
+ * Class to provide the graph data for the QuickstartExample
+ */
+public class QuickstartData {
+
+  /**
+   * Example Graph for QuickstartExample
+   *
+   * @see <a href="https://github.com/dbs-leipzig/gradoop/wiki/Getting-started">
+   * Gradoop Quickstart Example</a>
+   * @return example graph
+   */
+  public static String getGraphGDLString() {
+
+    return "" +
+      "g1:graph[\n" +
```

Comment:
Use the system line separator instead of \n ... maybe in most cases a break is not needed I think